target/linux/ath79/dts/qca9558_nec_aterm.dtsi

```diff
@@ -0,0 +1,379 @@
+			#address-cells = <1>;
+			#size-cells = <1>;
+
+			/*
+			 * since the OEM bootloader requires unknown
+			 * filesystem on firmware area, needs to be
+			 * replaced to u-boot before OpenWrt installation
+			 */
+			partition@0 {
+				label = "bootloader";
+				reg = <0x000000 0x020000>;
+			};
+
+			/* not compatible with u-boot */
+			partition@20000 {
+				label = "config";
+				reg = <0x020000 0x010000>;
+				read-only;
+
+				nvmem-layout {
+					compatible = "fixed-layout";
+					#address-cells = <1>;
+					#size-cells = <1>;
+
+					macaddr_config_0: macaddr@0 {
+						reg = <0x0 0x6>;
+					};
+
+					macaddr_config_6: macaddr@6 {
+						reg = <0x6 0x6>;
+					};
+
+					macaddr_config_c: macaddr@c {
+						reg = <0xc 0x6>;
+					};
+
+					macaddr_config_12: macaddr@12 {
+						reg = <0x12 0x6>;
+					};
+				};
+			};
+
+			partition@30000 {
+				label = "art";
+				reg = <0x030000 0x010000>;
+				read-only;
+
+				nvmem-layout {
+					compatible = "fixed-layout";
+					#address-cells = <1>;
+					#size-cells = <1>;
+
+					cal_art_1000: calibration@1000 {
+						reg = <0x1000 0x440>;
+					};
+
+					cal_art_5000: calibration@5000 {
+						reg = <0x5000 0x844>;
+					};
+				};
+			};
+
+			partition@40000 {
+				compatible = "denx,uimage";
+				label = "firmware";
+				reg = <0x040000 0xfc0000>;
+			};
+		};
+	};
+};
+
+&mdio0 {
+	status = "okay";
+
+	phy0: ethernet-phy@0 {
+		reg = <0>;
+
+		qca,ar8327-initvals = <
+			0x04 0x00080080 /* PORT0_PAD_MODE_CTRL */
+			0x08 0x00000000 /* PORT5_PAD_MODE_CTRL */
+			0x0c 0x07600000 /* PORT6_PAD_MODE_CTRL */
+			0x10 0xa12613a0 /* POWER_ON_STRAP */
+			0x50 0xcc36cc36 /* LED_CTRL0 */
+			0x54 0xca36ca36 /* LED_CTRL1 */
+			0x58 0xc936c936 /* LED_CTRL2 */
+			0x5c 0x03ffff00 /* LED_CTRL3 */
+			0x7c 0x0000007e /* PORT0_STATUS */
+			0x94 0x0000007e /* PORT6_STATUS */
+			0xe0 0xc74164d0 /* SGMII_CONTROL */
+		>;
+	};
+};
+
+&eth0 {
+	status = "okay";
+
+	nvmem-cells = <&macaddr_config_c>;
+	nvmem-cell-names = "mac-address";
+	phy-handle = <&phy0>;
+
+	gmac-config {
+		device = <&gmac>;
+
+		rgmii-enabled = <1>;
+		rxdv-delay = <2>;
+		rxd-delay = <2>;
+	};
+};
+
+&eth1 {
+	status = "okay";
+
+	nvmem-cells = <&macaddr_config_6>;
+	nvmem-cell-names = "mac-address";
+	pll-data = <0x03000000 0x00000101 0x00001616>;
+	qca955x-sgmii-fixup;
+
+	gmac-config {
+		device = <&gmac>;
+	};
+
+	fixed-link {
+		speed = <1000>;
+		full-duplex;
+	};
+};
+
+&gpio {
+	switch-reset {
+		gpio-hog;
+		gpios = <11 GPIO_ACTIVE_HIGH>;
+		output-high;
+	};
+};
+
+&pcie0 {
+	status = "okay";
+
+	wifi@0,0 {
+		compatible = "pci168c,003c";
+		reg = <0x0000 0 0 0 0>;
+		nvmem-cells = <&cal_art_5000>, <&macaddr_config_12>;
+		nvmem-cell-names = "calibration", "mac-address";
+	};
+};
+
+&pinmux {
+	/* mux GPIO19/20 as GPIO instead of native I2C on QCA9558 */
+	pmx_gpio_i2c_pins: gpio-i2c-pins {
+		pinctrl-single,bits = <0x10 0x0 0xff000000>,
+				      <0x14 0x0 0xff>;
+	};
+};
+
+&uart0 {
+	status = "okay";
```

Comment:
The default status is already set as “okay”.

Reply:
Oh, I didn't notice it. I'll remove...